HALO/specmacs/specMACS_polLL_cdsd.yaml

```diff
@@ -0,0 +1,1121 @@
+      auth: null
+      chunks: null
+      engine: netcdf4
+
+  HALO-0211_cb2:
+    description: Cloud droplet size distribution derived from polarimetric cloudbow measurements of liquid water clouds for circle_break 2 (cb2) on 2020-02-11
+    driver: opendap
+    args:
+      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/SPECMACS/CLOUD_DROPLET_SIZE_DISTRIBUTION/20200211/EUREC4A_HALO_specMACS-polLL_cdsd_20200211_HALO-0211_cb2_v1.0.nc
+      auth: null
+      chunks: null
+      engine: netcdf4
+
+  HALO-0211_cb3:
+    description: Cloud droplet size distribution derived from polarimetric cloudbow measurements of liquid water clouds for circle_break 3 (cb3) on 2020-02-11
+    driver: opendap
+    args:
+      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/SPECMACS/CLOUD_DROPLET_SIZE_DISTRIBUTION/20200211/EUREC4A_HALO_specMACS-polLL_cdsd_20200211_HALO-0211_cb3_v1.0.nc
+      auth: null
+      chunks: null
+      engine: netcdf4
+
+  HALO-0211_cb4:
+    description: Cloud droplet size distribution derived from polarimetric cloudbow measurements of liquid water clouds for circle_break 4 (cb4) on 2020-02-11
+    driver: opendap
+    args:
+      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/SPECMACS/CLOUD_DROPLET_SIZE_DISTRIBUTION/20200211/EUREC4A_HALO_specMACS-polLL_cdsd_20200211_HALO-0211_cb4_v1.0.nc
+      auth: null
+      chunks: null
+      engine: netcdf4
+
+  HALO-0211_rct1:
+    description: Cloud droplet size distribution derived from polarimetric cloudbow measurements of liquid water clouds for radar_calibration_tilted 1 (rct1) on 2020-02-11
+    driver: opendap
+    args:
+      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/SPECMACS/CLOUD_DROPLET_SIZE_DISTRIBUTION/20200211/EUREC4A_HALO_specMACS-polLL_cdsd_20200211_HALO-0211_rct1_v1.0.nc
+      auth: null
+      chunks: null
+      engine: netcdf4
+
+  HALO-0213_c3:
+    description: Cloud droplet size distribution derived from polarimetric cloudbow measurements of liquid water clouds for circle 3 (c3) on 2020-02-13
+    driver: opendap
+    args:
+      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/SPECMACS/CLOUD_DROPLET_SIZE_DISTRIBUTION/20200213/EUREC4A_HALO_specMACS-polLL_cdsd_20200213_HALO-0213_c3_v1.0.nc
+      auth: null
+      chunks: null
+      engine: netcdf4
+
+  HALO-0213_c4:
+    description: Cloud droplet size distribution derived from polarimetric cloudbow measurements of liquid water clouds for circle 4 (c4) on 2020-02-13
+    driver: opendap
+    args:
+      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/SPECMACS/CLOUD_DROPLET_SIZE_DISTRIBUTION/20200213/EUREC4A_HALO_specMACS-polLL_cdsd_20200213_HALO-0213_c4_v1.0.nc
+      auth: null
+      chunks: null
+      engine: netcdf4
+
```

Comment:
Is  HALO-0213_c5 missing on purpose?

Reply:
Thanks for mentioning this. I did not notice that this file was missing! There was indeed a problem with the processing of HALO-0213_c5 and, as I've just discovered, with 7 other segments as well. I am in the process of re-running the retrieval for these segments and will update the catalogue as soon as I am done.